Web_Development/FrontEnd/HTML_Text_Formatting.md

```diff
@@ -0,0 +1,49 @@
+# Documentation
+# HTML Text Formatting
+
+## HTML contains several elements or text for defining a special meaning
+## For example: We can format it into bold by using the ```<b>``` tag
+ ```<b>The Text is bold</b>```
+   Result: <b>The Text is bold</b>
+   
+ ## Similarly we can use other examples
+  1.For strong text:<br>
+  ```<strong>The Text is Strong</strong>```<br>Result: <strong>The Text is Strong</strong>
+  <br><br>
+  2. For italic text:<br>
+  ```<i>The Text is Strong</i>```<br>Result: <i>The Text is italic</i>
+  <br><br>
+  3.For emphasised text:<br>
+  ```<em>The Text is emphasised</em>```<br>Result: <em>The Text is emphasised</em>
+  <br><br>
+  4.For small text:<br>
+  ```<small>The Text is small</small>```<br>Result: <small>The Text is small</small>
+  <br><br>
+  5.For Subscript text:<br>
+  ```<strong>The Text is <sub>Subscript</sub></strong>```<br>Result: <strong>The Text is <sub>Subscript</sub></strong>
+  <br><br>
+  6.For Superscript text:<br>
+  ```<strong>The Text is <sup>Subscript</sup></strong>```<br>Result: <strong>The Text is <sup>Subscript</sup></strong>
```

Comment:
You should have the word "**superscript**" inside `<sup>` tags. That would make more sense to the readers.